roles/openshift_prometheus/tasks/oc_apply.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Checking generation of {{file_content.kind}} {{file_content.metadata.name}}
```

Comment:
certain api objects have their own modules and would replace this, yes. You should be able to see examples in the `openshift_logging_<component>` roles